modules/labs-react/aria-live/lib/AriaLive.Content.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import {colors, space} from '@workday/canvas-kit-react/tokens';
+import {
+  createComponent,
+  styled,
+  StyledType,
+  useModelContext,
+} from '@workday/canvas-kit-react/common';
+
+import {AriaLiveModelContext} from './AriaLive';
+import {AriaLiveModel} from './useAriaLiveModel';
+
+export interface AriaLiveContentProps {
+  model?: AriaLiveModel;
+  children: React.ReactNode;
+}
+
+const Container = styled('div')<StyledType>({
+  background: colors.frenchVanilla600,
+  padding: space.s,
+  border: `1px solid ${colors.licorice600}`,
+});
```

Comment:
Do we want the container to have a similar "off screen style" like https://github.com/AlmeroSteyn/react-aria-live/blob/master/src/modules/MessageBlock.js does?

Reply:
it most definitely should haha. this must be part of the default code or something, i'll update it now